python/mxnet/contrib/onnx/mx2onnx/_op_translations.py

```diff
@@ -1499,17 +1500,19 @@ def convert_slice_axis(node, **kwargs):
 
     axes = int(attrs.get("axis"))
     starts = int(attrs.get("begin"))
-    ends = int(attrs.get("end", None))
-    if not ends:
-        raise ValueError("Slice: ONNX doesnt't support 'None' in 'end' attribute")
+    ends = attrs.get("end", None)
+    if not ends or ends == 'None':
+        # ONNX doesn't support None for ends. Since ends=None depicts
```

Comment:
why should None be mapped to INT_MAX?

Reply:
I think it is sufficient to map None to the dimension of the particular axis. Changing this in the next iteration.

Reply:
@anirudhacharya fixed this. the latest commit does not map None to INT_MAX